system/assets/js/admin/presidecore/preside.datetimepicker.js

```diff
@@ -0,0 +1,100 @@
+( function( $ ){
+
+	$('.datetimepicker').each( function() {
+		var $thisPicker      = $( this )
+		  , pickerConfig     = $thisPicker.data()
+		  , relativeToField  = pickerConfig.relativeToField
+		  , relativeOperator = pickerConfig.relativeOperator
+		  , conf, $form, relativeField, datetimePicker;
+
+		conf = {
+			  minDate   : pickerConfig.startDate || false
+			, maxDate   : pickerConfig.endDate   || false
+			, icons     : {
+                  time:     'fa fa-clock-o'
```

Comment:
Could you fix the indentation / whitespace here? Seems to go awry in the icons section.